tests/apollo/test_skvbc_chaotic_startup.py

```diff
@@ -303,6 +303,52 @@ async def test_f_minus_one_staggered_replicas_requesting_vc(self, bft_network, s
 
         await self._wait_for_replicas_to_generate_checkpoint(bft_network, skvbc, expected_next_primary, bft_network.all_replicas(without={initial_primary}))
 
+    @with_trio
+    @with_bft_network(start_replica_cmd_with_vc_timeout("20000"),
+                      selected_configs=lambda n, f, c: n == 7)
+    @with_constant_load
+    async def stuck_view_change_bug_recreation(self, bft_network, skvbc, nursery):
```

Comment:
if you remove the fix, does the test fail? 😄

Reply:
yes, I also had put logs locally to verify the execution goes through the newly implemented parts after the fix :)